simulator/protocols/simple/SimpleProtocolGround.py

```diff
@@ -14,17 +14,20 @@ def initialize(self, stage: int):
     def handle_timer(self, timer: dict):
         pass
 
-    def handle_packet(self, message: dict):
-        print(f"SimpleProtocolGround packets: {self.packets}, {message['sender']}")
-        if message["sender"] == SenderType.DRONE:
-            print(f"SimpleProtocolGround packets2: {self.packets}, {message['sender']}")
-            self.packets += message['content']
+    def handle_packet(self, message: str):

```

Comment:
Please change the function signature in IProtocol to force all protocols to use strings as their messages